### ISSUE TITLE
Implement Loss Functions

### ISSUE THREAD

**Issue by Shashwatpandey4** [Shashwatpandey4/Cortex#6]
Core loss functions:
- [ ] MSE Loss
- [ ] Cross Entropy Loss
- [ ] Binary Cross Entropy
- [ ] Custom loss function support
